feature-libs/user/account/components/verification-token-form/verification-token-dialog.component.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component } from '@angular/core';
+import { FocusConfig, LaunchDialogService } from '@spartacus/storefront';
+
+export enum VERIFICATION_TOKEN_DIALOG_ACTION {
```

Comment:
Can we move this enum outside the component file? It should be part of the model file.

Reply:
fixed